pkg/sql/exec/error.go

```diff
@@ -50,8 +51,11 @@ func CatchVectorizedRuntimeError(operation func()) (retErr error) {
 				panic(fmt.Sprintf("panic line %q not found in the stack trace\n%s", panicLineSubstring, stackTrace))
 			}
 			if scanner.Scan() {
-				if strings.HasPrefix(strings.TrimSpace(scanner.Text()), execPackagePrefix) {
-					// We only want to catch runtime errors coming from the exec package.
+				panicEmittedFrom := strings.TrimSpace(scanner.Text())
+				if strings.HasPrefix(panicEmittedFrom, execPackagePrefix) ||
```

Comment:
Consider extracting this into a function

Reply:
Done.